backend/app/migrations/versions/4ba3479cb8df_create_user_table_and_roles.py

```diff
@@ -0,0 +1,50 @@
+"""create user table and roles
```

Comment:
just wanted to confirm all files under migration folder are autogenerated from defined model/schemas correct

Reply:
Yes for the most part. The only exception would be seeding for constants or roles in this case.